src/SettingStore.php

```diff
@@ -237,18 +192,13 @@ public function load($force = false)
 	}
 
 	/**
-	 * Read data from a store or cache
+	 * Read data from a store.
 	 *
 	 * @return array
 	 */
 	private function readData()
```

Comment:
It always calls this version of the method because it is private. It should be protected here and in CachedSettingStore.